tensorflow/g3doc/tutorials/deep_cnn/index.md

```diff
@@ -39,7 +39,8 @@ https://en.wikipedia.org/wiki/Rectifier_(neural_networks))), [max pooling](
 ../../api_docs/python/nn.md#max_pool) ([wiki](
 https://en.wikipedia.org/wiki/Convolutional_neural_network#Pooling_layer))
 and [local response normalization](
-../../api_docs/python/nn.md#local_response_normalization) ([quora](https://www.quora.com/What-is-local-response-normalization)).
+../../api_docs/python/nn.md#local_response_normalization) 
+(Chapter 3.3 in [AlexNet paper](http://www.cs.toronto.edu/~fritz/absps/imagenet.pdf)).
```

Comment:
http://papers.nips.cc/paper/4824-imagenet-classification-with-deep-convolutional-neural-networks.pdf feels like a more official link, otherwise, looks good!